### PR TITLE
5252 - Add improvements to the Expandable Cards

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR improves the expandable cards without any changes to the old one. 

Has ability to:
- rotate the button actions vertically
- use expandable cards (You can still use the old implementation)

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5252

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/cards/example-expandable-cards.html
- Click the header, it should expand the card pane
- Check the look and feel and compare to the design

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: src/components/cards/cards.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  init() {
+    this
+      .setup()
+      .build()
+      .handleEvents();
+  },
+
+  setup() {
+    this.id = this.element.attr('id');
+    if (!this.id || this.id === undefined) {
```
Comment (tmcconechy) [inline src/components/cards/cards.js:47]: We could add the utils.addAttributes here as it will be needed for id's

**Review by tmcconechy — CHANGES_REQUESTED** [review]
This looks good only a few PR suggestions

1) Add testing id's api
2) When clicking the buttons im not seeing the "pulse" animation. I see one button is missing the classes so that could be the issue 
```
<button type="button">
                        <svg class="icon" focusable="false" aria-hidden="true" role="presentation">
                          <use href="#icon-caret-right"></use>
                        </svg>
                    </button>
```
3) Could also add a couple tests , maybe just one e2e with the expander open?
4) The text seems to sit lower than the design
Screen Shot 2021-06-10 at 10 08 46 AM

**Comment by ericangeles** [comment]
@tmcconechy just waiting for your feedback before I add some e2e. :)

**Review by EdwardCoyle — APPROVED** [review]
@ericangeles Nicely done 👍

**Comment by tmcconechy** [comment]
Looks Great!

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**